src/features/pass-survey/model/tests/ScoresExtractor.test.ts

```diff
@@ -0,0 +1,471 @@
+        {
+          alert: null,
+          color: null,
+          id: 'mock-item-id-100',
+          image: null,
+          isHidden: false,
+          score: 100,
+          text: 'mock-text-100',
+          tooltip: null,
+          value: 0,
+          isNoneOption: false,
+        },
+        {
+          alert: null,
+          color: null,
+          id: 'mock-item-id-200',
+          image: null,
+          isHidden: false,
+          score: 200,
+          text: 'mock-text-200',
+          tooltip: null,
+          value: 1,
+          isNoneOption: false,
+        },
+        {
+          alert: null,
+          color: null,
+          id: 'mock-item-id-300',
+          image: null,
+          isHidden: false,
+          score: 300,
+          text: 'mock-text-300',
+          tooltip: null,
+          value: 2,
+          isNoneOption: false,
+        },
+      ],
+    },
+  };
+  return checkboxesItem;
+};
+
+const getSliderItem = (): SliderPipelineItem => {
+  const sliderItem: SliderPipelineItem = {
+    type: 'Slider',
+    timer: null,
+    name: 'item-slider',
+    payload: {
+      alerts: [],
+      isContinuousSlider: false,
+      leftImageUrl: null,
+      rightImageUrl: null,
+      leftTitle: null,
+      rightTitle: null,
+      minValue: 5,
+      maxValue: 10,
+      scores: [5, 6, 7, 8, 9, 10],
+      showTickLabels: false,
+      showTickMarks: false,
+    },
+  };
+  return sliderItem;
+};
+
+describe('ScoresCalculator: test collectScoreForRadio', () => {
+  let extractor: ScoresExtractor;
+
+  beforeEach(() => {
+    extractor = new ScoresExtractor(
+      new ScoreConditionsEvaluator(),
+      new ScoresCalculator(),
+      {
+        log: jest.fn(),
+        warn: jest.fn(),
+        error: jest.fn(),
+        info: jest.fn(),
+      } as unknown as ILogger,
+    );
+  });
+
+  [
+    { expectedValue: 517, calculationType: 'sum' },
+    { expectedValue: 517 / 3, calculationType: 'average' },
+    { expectedValue: 80.78125, calculationType: 'percentage' },
+  ].forEach(({ expectedValue, calculationType }) => {
+    it(`Should return ${expectedValue} when calculationType is ${calculationType} and no set conditions`, () => {
+      const pipelineItems: PipelineItem[] = [];
+
+      const radiosItem: PipelineItem = getRadiosItem();
+
+      const checkBoxesItem: PipelineItem = getCheckboxesItem();
+
+      const sliderItem: PipelineItem = getSliderItem();
+
+      pipelineItems.push(radiosItem);
+      pipelineItems.push(checkBoxesItem);
+      pipelineItems.push(sliderItem);
+
+      const radioAnswer: RadioResponse = radiosItem.payload.options[0];
+
+      const checkBoxAnswer: Item[] = [
+        checkBoxesItem.payload.options[1],
+        checkBoxesItem.payload.options[2],
+      ];
+
+      const sliderAnswer: SliderResponse = 7;
+
+      const answers: Answers = {
+        0: { answer: radioAnswer },
+        1: {
+          answer: checkBoxAnswer,
+        },
+        2: { answer: sliderAnswer },
+      };
```

Comment:
I'm not sure I see the benefit of having this section (lines 167-194) copy-pasted for every test (it's the exact same chunk of code used 4 times); couldn't `pipelineItems` and `answers` also just be defined as static top-level constants, reused by all the tests?

Reduced redundancy would make the code easier to read and maintain.